angular/cmbs-property-summary/src/app/property-locations/property-locations.component.ts

```diff
@@ -8,6 +8,7 @@ import { Observable } from "rxjs/Observable";
 import { CollectionViewer } from "@angular/cdk/collections";
 import { DataSource } from "@angular/cdk/collections";
 
+declare const google;
```

Comment:
I think the google global needs to be a variable rather than a const.  All the examples I found did something like this:
declare let google:any;
I suspect a const is going to fail if you attempt to update the map